lib/licensed/sources/swift.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+require "json"
+require "pathname"
+require "uri"
+
+require "licensed/sources/helpers/content_versioning"
+
```

Comment:
This require isn't needed since you're no longer including the content versioning module

```suggestion
```